lib/api/holdings_api.rb

```diff
@@ -30,11 +30,13 @@ class HoldingsAPI < Sinatra::Base
     access_count = overlap_records.map(&:access_count).reduce(:+)
     copy_count = overlap_records.map(&:copy_count).reduce(:+)
     currently_held_count = overlap_records.map(&:current_holding_count).reduce(:+)
+    deposited = (ht_item.billing_entity == billing_entity(organization)) ? 1 : 0
```

Comment:
After pondering I think we want:

`mapto_inst_id(ht_item.billing_entity) == mapto_inst_id(organization)`